resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,86 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('has_disabled_link_warning') === 'true';
         }
     }"
-    class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
+    <a
+        :href="hasDisabledLinkWarning() ? '{{ $url }}' : 'javascript:;'"
```

Comment:
this seems to work, except that it doesn't reload while you stay on the page itself. After a reload it shows the url properly though

Reply:
Yeah noticed too, may have an idea about it I'll check soon 🤔

Reply:
Did tried a couple of things but meh, so still in the same state for that point, but rest is good now.